modules/core/src/main/scala/org/scalasteward/core/forge/github/GitHubReviewers.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018-2023 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.forge.github
+
+import io.circe.Encoder
+import io.circe.generic.semiauto.deriveEncoder
+
+case class GitHubReviewers(reviewers: List[String])
+
+object GitHubReviewers {
+  implicit val gitHubReviewersEncoder: Encoder[GitHubReviewers] = deriveEncoder
```

Comment:
**thought:** take into account that this won't work for requesting team reviews, since those are done through a different parameter.

See docs: https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#request-reviewers-for-a-pull-request

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
Thank you for pointing out. Yes I am ready to add this feature as well - either in the current PR or in new one (if the solution will be an addition that won't require to break existing code for assignees/reviewers I feel that it will be easier to separate it as another PR, if we need to change something in the new code - of course we should do it here).

So, the straightforward solution will be to just add `teamReviewers` parameter to configuration but the downside is that it will not be functional for all implementations but GitHub. It's the same situation as for example AzureRepos does not have "assignee" feature, so, should not be a big deal.

What do you think about implementing `teamReviewers` this way?

Reply:
Maybe adding it so teams are added like: `team:my-org/my-team` so it's easy to pattern-match. It will also be easy to just strip that from forges where both are added the same way.

Reply:
Or maybe just add them as `my-org/my-team` and pattern-match on GitHub forge for `$org/$team` and use `teamReviewers` for those that match. WDYT?

Reply:
`my-org/my-team` looks neat, I like it!

I only wonder how to make sure that such entries are filtered for implementations other than GitHub? Should we have some common function that will filter out team reviewers in `ForgeApi` implementations, or we should make it on the level above and pass team reviewers as `ForgeSpecificCfg` for GiHub only?

Reply:
We should not need to filter for other implementations (given this is a forge-specific), on GitHub we can just pattern-match on the pattern. On other forges where there is no distinction we will just use the same string that is added to the config file.

Reply:
Ah sure, you are right! Thank you

The change seems not very complex then, I will add it to the current PR

Reply:
I've added team reviewers support with `org/team` construct, please take a look